demo/demo.js

```diff
@@ -20,15 +20,36 @@ window.addEventListener('DOMContentLoaded', function () {
 			message = 'You did not enter a message...';
 		}
 
+		//create the text version of the command for displaying
+		let text = `
+		<label> JS Command used to create this notification:</label>
+		<hr>
+<pre>
+window.createNotification({
+	closeOnClick: ${closeOnClick},
+	displayCloseButton: ${displayClose},
+	positionClass: ${position},
+	showDuration: Number(${duration}),
+	theme: ${theme}
+})({
+	title: ${title},
+	message: ${message}
+});
+</pre>
+		`;
+
+		document.getElementById("cmdoutput").innerHTML =text;

```

Comment:
ESLint requires single quote for this one
```suggestion
		document.getElementById('cmdoutput').innerHTML = text;

```